wire-swift-generator/src/main/java/com/squareup/wire/swift/SwiftGenerator.kt

```diff
@@ -1192,17 +1254,21 @@ class SwiftGenerator private constructor(
 
     if (type.extensionFields.isNotEmpty()) {
       val extendedFieldsExtension = ExtensionSpec.builder(structType)
-        .addDoc("Extensions of %T\n", structType)
         .apply {
-          type.extensionFields.filter { !it.isMap }.forEach { field ->
+          if (!forStorageType) {
+            addDoc("Extensions of %T\n", structType)
+          }
+          type.extensionFields.forEach { field ->
             val property = PropertySpec.varBuilder(field.safeName, field.typeName, PUBLIC)
               .apply {
-                if (field.documentation.isNotBlank()) {
+                if (!forStorageType && field.documentation.isNotBlank()) {
                   addDoc("\n%L\n", field.documentation.sanitizeDoc())
                 }
-                addDoc("\nSource: %L\n", field.location.withPathOnly())
+                if (!forStorageType) {
+                  addDoc("\nSource: %L\n", field.location.withPathOnly())
+                }
 
-                if (field.isDeprecated) {
+                if (!forStorageType && field.isDeprecated) {
```

Comment:
I _think_ we want to mark as deprecated regardless?

Reply:
we are in the other method for generating storage delegates?

Reply:
nm spoke offline we want in both and updated.